lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -247,7 +247,13 @@ sub list_running_ajax {
                 progress => $job->progress_info,
             });
     }
-    $self->render(json => {data => \@running});
+    my %response = (data => \@running);
+    my $max_running = OpenQA::App->singleton->config->{scheduler}->{max_running_jobs};
+    if ($max_running >= 0) {
+        my $running = $self->schema->resultset('Jobs')->count({state => [OpenQA::Jobs::Constants::EXECUTION_STATES]});
+        $response{max_running_jobs} = $max_running if $running >= $max_running;
+    }
```

Comment:
I'd avoid the additional database query. We have just populated an array of running jobs so let's just use its size:
```suggestion
    $response{max_running_jobs} = @running if $max_running >= 0 && @running >= $max_running;
```

Reply:
Like I wrote in the commit message, I can't use that value because you can limit the query, e.g. `/tests?match=extra`.
Otherwise I would also avoid the query.

Reply:
Mh… that makes sense. It is nevertheless bad that we have to do a count on a huge table and a column we don't have an index for. I guess it is ok considering the code path is only entered when the setting is configured.

Reply:
We talked about it, and while `state` is indexed, we only want the notice on a page without any filters (otherwise it would be confusing), and in this case using the existing `@running` variable does the job.
Pushed.